test/others/minimax.spec.js

```diff
@@ -0,0 +1,170 @@
+var minimax = require('../../src/others/minimax.js').minimax;
+
+describe('Minimax with tic tac toe', function () {
```

Comment:
That's a fantastic example! As you suggested in your comment, it's leaning a bit towards an integration test.

I think, having a simpler game is fine as well. This way we won't need that much logic to setup our tests and they'll be less error prone. Consider a game where each next step is a binary decision, and the leafs of the binary tree determine which player is the winner.

I really like this example, so I'd love to keep it, and also add another test group which tests the `minimax` example with a simpler binary game.

Reply:
I'm glad you like it! 
Sounds reasonable. I'm not sure if I follow the idea of the simpler game. What do you mean by `the leafs of the binary tree determine which player is the winner`? Should I create some binary tree with one "winning leaf"?

Reply:
It doesn't have to be a real game, it can be any game. My proposal is for something like this:

```
     o
   /    \
  o      o
 /  \   /  \
1   -1 1   -1
```

If we get `1` player "A" wins, otherwise, if we get `-1` player "B" wins. It's pretty much pre-defined, depends on who would start first. It'll be easier to reason about IMO.

Reply:
I think I got it. Indeed, it's much simpler.